rest-service/manager_rest/deployment_update/handlers.py

```diff
@@ -184,6 +186,43 @@ def modify(self, ctx, current_entities):
                 self._modify_relationship_operation,
                 self._modify_node_operation)
 
+    def _update_stored_operations(self, ctx, node, new_operation):
+        """Update the operations table with the new operation inputs."""
+
+        def _filter_operation(column):
+            # path in the parameters dict that stores the node name
+            node_name_path = ('task_kwargs', 'cloudify_context', 'node_name')
+            # this will use postgres' json operators
+            return cast(column, JSON)[node_name_path].astext == node.id
+
+        if not ctx.deployment_update.update_executions:
```

Comment:
I am not sure that 'update_executions' is really clear, because if don't know what it does you can think that it could do more.